### PR TITLE
Swope rosa hcp

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: clusters/release_notes/mce_known_issues.adoc [hunk 1]
```diff
@@ -25,13 +25,76 @@ Review the known issues for Cluster lifecycle with {mce-short}. The following li
 
 Cluster management known issues and limitations are part of the Cluster lifecycle with {mce-short} documentation.
 
+* <<cluster-issues-install,Installation>>
 * <<cluster-issues-mce,Cluster management>>
 * <<cluster-issues-cim,Central infrastructure management>>
 * <<cluster-issues-msa,Managed Service Account add-on>>
+//??
```
Comment (swopebe) [inline clusters/release_notes/mce_known_issues.adoc:32]: I can address this in my larger release note review in a different issue.

File: clusters/support_troubleshooting/trouble_unknown_status_rosa_hcp.adoc [hunk 2]
```diff
@@ -0,0 +1,58 @@
+[#trouble-unknown-status-rosa-hcp]
```
Comment (swopebe) [inline clusters/support_troubleshooting/trouble_unknown_status_rosa_hcp.adoc:1]: Please review this file.

File: clusters/release_notes/mce_known_issues.adoc [hunk 3]
```diff
@@ -25,13 +25,76 @@ Review the known issues for Cluster lifecycle with {mce-short}. The following li
...
+
```
Comment (swopebe) [inline clusters/release_notes/mce_known_issues.adoc:33]: Please review this file.

File: clusters/support_troubleshooting/trouble_unknown_status_rosa_hcp.adoc [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+= Troubleshooting managed clusters _Unknown_ on {rosa} with hosted control planes cluster
```
Comment (oafischer) [inline clusters/support_troubleshooting/trouble_unknown_status_rosa_hcp.adoc:2]: Should _Unknown_ be `Unknown` like it is in the paragraph below?

File: clusters/release_notes/mce_known_issues.adoc [hunk 5]
```diff
@@ -25,13 +25,76 @@ Review the known issues for Cluster lifecycle with {mce-short}. The following li
...
+[#cluster-issues-install]
+== Installation 
+
+Learn about known issues and limitations during {mce-short} installation.
+
+=== Status stuck when installing on {rosa} with hosted control plane cluster
```
Comment (oafischer) [inline clusters/release_notes/mce_known_issues.adoc:40]: There is an issue with the formatting here that affects the rest of the file

File: clusters/support_troubleshooting/trouble_unknown_status_rosa_hcp.adoc [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+The status of all managed clusters on a {rosa} hosted clusters suddenly becomes `Unknown`.
+
+[#symptom-unknown-status-rosa-hcp]
+== Symptom: All managed clusters are in _Unknown_ status on {rosa} with hosted control planes cluster
+
+When you check the `klusterlet-agent` pod log in the `open-cluster-management-agent` namespace on your managed cluster, you see an error that resembles the following:
+
+----
```
Comment (oafischer) [inline clusters/support_troubleshooting/trouble_unknown_status_rosa_hcp.adoc:12]: Add source highlighting here

File: clusters/release_notes/mce_known_issues.adoc [hunk 7]
```diff
@@ -25,13 +25,76 @@ Review the known issues for Cluster lifecycle with {mce-short}. The following li
...
+
+Installation status might get stuck in the `Installing` state when you install {mce-short} on a {rosa} with hosted control planes cluster. The `local-cluster` might also remain in the `Unknown` state. 
+
+When you check the `klusterlet-agent` pod log in the `open-cluster-management-agent` namespace on your hub cluster, you see an error that resembles the following:
+
++
+[source,terminal]
```
Comment (oafischer) [inline clusters/release_notes/mce_known_issues.adoc:47]: I'm personally OK with using terminal for error messages, but do we want to stick with our previous rule of consistency (only use bash) or do we want to revisit that and have different highlighting for different type of content?

File: clusters/support_troubleshooting/trouble_unknown_status_rosa_hcp.adoc (hunk 4 above)
Comment (swopebe) [reply clusters/support_troubleshooting/trouble_unknown_status_rosa_hcp.adoc:2]: Yes, but not in the title bc we don't code the titles; I don't remember us changing that?

File: clusters/release_notes/mce_known_issues.adoc (hunk 7 above)
Comment (swopebe) [reply clusters/release_notes/mce_known_issues.adoc:47]: I didn't use bash bc you didn't want to use it; Alexandra is using terminal in siteconfig and I don't think it matters much...we are losing that when we change tools anyway.

File: clusters/support_troubleshooting/trouble_unknown_status_rosa_hcp.adoc (hunk 4 above)
Comment (oafischer) [reply clusters/support_troubleshooting/trouble_unknown_status_rosa_hcp.adoc:2]: You are right I was thinking about conrefs

File: clusters/support_troubleshooting/trouble_unknown_status_rosa_hcp.adoc (hunk 6 above)
Comment (swopebe) [reply clusters/support_troubleshooting/trouble_unknown_status_rosa_hcp.adoc:12]: done

File: clusters/release_notes/mce_known_issues.adoc (hunk 5 above)
Comment (swopebe) [reply clusters/release_notes/mce_known_issues.adoc:40]: The message I indented at first, but we don't need it and it wasn't rendering anyway. Removed.

**Review by oafischer — APPROVED** [review]

**Pull request merged**